spinoso-string/src/lib.rs

```diff
@@ -1684,6 +1705,89 @@ impl String {
         inner(self.inner.as_slice(), needle, offset)
     }
 
+    /// Returns the Integer byte-based index of the first occurrence of the
+    /// given substring.
+    ///
+    /// Returns [`None`] if not found. If the second parameter is present, it
+    /// specifies the position in the string to begin the search.
```

Comment:
Ditto. Can we clarify byte ha char position?

Reply:
fixed in d8eda7b7fe5eba53c460b4d80c28694da2905fe6